### PR TITLE
make landing light actually work

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request

https://github.com/tgstation/TerraGov-Marine-Corps/assets/98577688/ebbdef0e-277d-4081-88fb-410dcecfbbd6
## Why It's Good For The Game
they work now
## Changelog
:cl:
fix: landing lights now work
/:cl:

### CODE REVIEW

**Review by TiviPlus — APPROVED** [review]

**Comment by spookydonut** [comment]
I cant believe this shitty coder sprite i made is still around

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -687,65 +687,48 @@
...
-/obj/machinery/landinglight/ds1/delaytwo/turn_on()
-	icon_state = "landingstripe2"
-	set_light(2,2)
-
-/obj/machinery/landinglight/ds1/delaythree/turn_on()
-	icon_state = "landingstripe3"
-	set_light(2,2)
-
-/obj/machinery/landinglight/ds2/delayone/turn_on()
-	icon_state = "landingstripe1"
```
Comment (spookydonut) [inline code/modules/power/lighting.dm:739]: it looks way nicer if you use the delay states too.

**Comment by ivanmixo** [comment]
What's with the fat diff? Please tell me you didn't use dreammaker

**Comment by tyeagg** [comment]
> What's with the fat diff? Please tell me you didn't use dreammaker

its literally just map edits

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**